base/multi.jl

```diff
@@ -1514,101 +1514,17 @@ macro everywhere(ex)
     end
 end
 
+#FIXME delete?
 function pmap_static(f, lsts...)
```

Comment:
This was added in https://github.com/JuliaLang/julia/commit/b325999e11a0e1a04eba46690e3ffbf5a6e06051 but since it has not been exported, nor used anywhere in Base, I guess it is safe to remove.

The functionality though, i.e., static partitioning of the work load across workers can be achieved in the new `pmap` via batching, right? Maybe provide a keyword arg `partition=fixed` to expose it?